docs/proposals/20240807-in-place-updates.md

```diff
@@ -875,18 +874,17 @@ Since the fallback to machine replacement is enabled, the MachineDeployment cont
 ##### `CanUpdateMachine` endpoint
 ##### Request
 > Requirements:
-> * Current Machine Spec
-> * Desired Machine Spec, including bootstrap and and infra machine templates
+> * Desired Machine/Bootstrap/InfraMachine changes
 
 ##### Response
 > Requirements:
-> * Set of supported changes, probably and array of strings (the path in the object)
+> * Set of supported changes, probably an array of strings (the path in the object)
 > * Error
 
 ##### `UpdateMachine` endpoint
 ##### Request
 > Requirements:
-> * Machine reference
+> *  Machine Spec, including bootstrap and and infra machine templates
```

Comment:
why do we need to include the machine spec? isn't it enough to include the machine's ref (name and namespace)?
the the updater can read it plus the bootstrap and infra machine template